pkgs/top-level/all-packages.nix

```diff
@@ -29806,6 +29806,8 @@ with pkgs;
 
   polytopes_db = callPackage ../data/misc/polytopes_db { };
 
+  poptracker = callPackage ../applications/misc/poptracker { };
+
```

Comment:
This should be removed and then the package should be moved to the `pkgs/by-name/so/some-package/package.nix` hierarchy as this is the preferred way to add new packages (:

```suggestion
```